include/TrigScint/TrigScintQIEDigiProducer.h

```diff
@@ -0,0 +1,132 @@
+/**
+ * @file TrigScintQIEDigiProducer.h
+ * @brief Class that simulates QIE chip of the trigger scintillator
+ * @author Niramay Gogate, Texas Tech University
+ */
+
+#ifndef EVENTPROC_TRIGSCINTQIEDIGIPRODUCER_H
+#define EVENTPROC_TRIGSCINTQIEDIGIPRODUCER_H
+
+/*~~~~~~~~~~~~~~~~*/
+/*   C++ StdLib   */
+/*~~~~~~~~~~~~~~~~*/
+#include <time.h>
+
+/*~~~~~~~~~~*/
+/*   ROOT   */
+/*~~~~~~~~~~*/
+#include "TRandom3.h"
+
+// LDMX
+#include "DetDescr/TrigScintID.h"
+#include "TrigScint/Event/TrigScintHit.h"
+#include "SimCore/Event/SimCalorimeterHit.h"
+
+/*~~~~~~~~~~~~~~~*/
+/*   Framework   */
+/*~~~~~~~~~~~~~~~*/
+#include "Framework/EventProcessor.h"
+#include "Framework/Configure/Parameters.h"
+
+// QIE output class
+#include "TrigScint/Event/TrigScintQIEDigis.h"
+#include "TrigScint/SimQIE.h"
+
+namespace ldmx {
```

Comment:
Same namespace question as below: when should we make the move to `trigscint` rather than the `ldmx` namespace?

Reply:
this has now been resolved in the latest commit.  Producers are now all in trigscint namespace and data types are in ldmx.